components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -8,8 +8,8 @@
 #include "brave/common/brave_paths.h"
 #include "brave/components/brave_rewards/browser/rewards_service_impl.h"
 #include "brave/components/brave_rewards/browser/rewards_service_factory.h"
-#include "brave/vendor/bat-native-ledger/src/bat_helper.h"
-#include "brave/vendor/bat-native-ledger/src/static_values.h"
+#include "brave/vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.h"
```

Comment:
we shouldn't ever be using `brave/vendor/bat-native-ledger/src/` in the header path. The test target should have `bat-native-ledger:internal_config` so it has the correct include paths. Using the full path like this always indicates that you have a gn issue

Reply:
fixed